core/templates/dev/head/pages/exploration-editor-page/services/learner-answer-details-data.service.spec.ts

```diff
@@ -86,36 +75,92 @@ describe('Learner answer details service', function() {
     $httpBackend.verifyNoOutstandingRequest();
   });
 
-  it('should successfully fetch learner answer info data from the backend',
-    function() {
-      var successHandler = jasmine.createSpy('success');
-      var failHandler = jasmine.createSpy('fail');
+  describe('.fetchLearnerAnswerInfoData', function() {
+    beforeEach(function() {
+      sampleDataResults = {
+        learner_answer_info_data: [{
+          state_name: 'fakeStateName',
+          interaction_id: 'fakeInteractionId',
+          customization_args: 'fakeCustomizationArgs',
+        }]
+      };
+    });
 
-      $httpBackend.expect('GET', '/learneranswerinfohandler/' +
-        'learner_answer_details/exploration/12345').respond(sampleDataResults);
-      LearnerAnswerDetailsDataService.fetchLearnerAnswerInfoData().then(
-        successHandler, failHandler);
-      $httpBackend.flush();
+    it('should successfully fetch learner answer info data from the backend',
```

Comment:
Can we have a spy for the createFromBackendDicts and ensure that it was called in this case.

Reply:
Done